setup.cfg

```diff
@@ -39,8 +39,10 @@ classifiers =
 packages = find:
 install_requires =
     future >=0.14
+    cython
     packaging >=15.0
     numpy >=1.10,!=1.14.0,!=1.14.1,!=1.14.2
+    pyNFFT @ git+ssh://git@github.com/ghisvail/pyNFFT.git@e2da0af#egg=pyNFFT
```

Comment:
Please don't make this a core dependency. It should be optional. Same goes for Cython.

Reply:
so should I just have it in travis and add in the docs some instruction on how to install the library? (also some checks to verify that the library is installed when using the classes)

Reply:
Indeed. Please have a look at how this has been done for Astra to get an idea.

Reply:
In `setup.cfg` you can add them to the [`all`](https://github.com/odlgroup/odl/blob/5b576d700d902ce8ce6528bd86a65166d403a43d/setup.cfg#L67-L74) part of `[options.extras_require]`.
In Travis, yes, running unittests for NFFT is desirable. Just explicitly install the stuff you need.

For runtime checking, you can pretty much clone the `PYFFTW_AVAILABLE` section.

Instructions in the docs would be nice. (Note that there's also a package on `conda-forge`, that should probably the go-to solution for user installation. With pyFFTW we've also had build issues in the past and moved away from `pip` on Travis.)

Reply:
I am having some trouble using conda for now on my local machine, so I can't really test the local user installation. For now I have put the solution using `pip` but if you really want me to dig into the forge solution locally I can try and do it (it works on travis so it should locally as well).